teaching-platform/models.py

```diff
@@ -0,0 +1,36 @@
+from enum import Enum
+
+from extensions import db
+
+
+class User(db.Model):
+    __tablename__ = "users"
+    id = db.Column(db.Integer, primary_key = True)
+    username = db.Column(db.String(20), unique=True, nullable= False)
+    password = db.Column(db.String(90),nullable= False)
+    role = db.Column(db.String(20),nullable= False)
+
+    def __repr__(self):
+        return f"User(id: {self.id}, username: {self.username}, role: {self.role})"
+
+
+#this model is needed for showing only public data not personal informations like password, for exaple in see_all_users()
+class CensuredUser:
+    def __init__(self, id, username, role):
+        self.id = id
+        self.username = username
+        self.role = role
+
+#STUDENT, TEACHER, ADMIN
+def get_all_roles():
```

Comment:
I suggest something more robust eg.
```python
return [role.value for role in Role]
```
That way when you add new Role in your enum and forget to update this function - it will still work, since roles here are not hardcoded.

Reply:
Added